app/models/common_application.rb

```diff
@@ -15,6 +15,8 @@ class CommonApplication < ApplicationRecord
   enum previously_received_assistance: { unfilled: 0, yes: 1, no: 2 },
        _prefix: :previously_received_assistance
 
+  enum living_situation: { unknown_living_situation: 0, stable_address: 1, temporary_address: 2, homeless: 3 }
```

Comment:
I didn't add a `_prefix` here, as I felt the usage was better without it:

```
current_app.unknown_living_situation?
current_app.stable_address?
current_app.permanent_address?
current_app.homeless?
```

The downside I see is that we don't have a consistent usage with other enums.

Reply:
Makes sense to me. Reads well. If we feel pain around the inconsistency later, we can easily refactor!